src/Functions/FunctionJoinGet.h

```diff
@@ -77,13 +77,15 @@ class JoinGetOverloadResolver final : public IFunctionOverloadResolverImpl
     String getName() const override { return name; }
 
     FunctionBaseImplPtr build(const ColumnsWithTypeAndName & arguments, const DataTypePtr &) const override;
-    DataTypePtr getReturnType(const ColumnsWithTypeAndName & arguments) const override;
+    DataTypePtr getReturnType(const ColumnsWithTypeAndName &) const override { return {}; }
 
     bool useDefaultImplementationForNulls() const override { return false; }
     bool useDefaultImplementationForLowCardinalityColumns() const override { return true; }
 
     bool isVariadic() const override { return true; }
     size_t getNumberOfArguments() const override { return 0; }
+    ColumnNumbers getArgumentsThatAreAlwaysConstant() const override { return {2}; }
```

Comment:
It means that the 3rd argument is always constant - does not look correct.
Did you mean - first two arguments (`{0, 1}`)?

Reply:
Yeah, I'll update...